tests/MongoDB.Driver.Examples/InsertDataWithEncryptedFieldsWithAutoEncryption.cs

```diff
@@ -0,0 +1,295 @@
+            var nameQuery = Builders<BsonDocument>.Filter.Eq("name", __sampleNameValue);
+            var normalClientNameResult = collection.Find(nameQuery).FirstOrDefault();
+            if (normalClientNameResult == null)
+            {
+                throw new Exception("Assert that the filtered data has been found.");
+            }
+
+            Console.WriteLine($"Query by name returned the following document:\n {normalClientNameResult}.");
+        }
+
+        private IMongoClient CreateEncryptedClient(
+            CollectionNamespace medicalRecordsNamespace,
+            CollectionNamespace keyVaultNamespace,
+            BsonDocument schema)
+        {
+            var kmsProviders = new Dictionary<string, IReadOnlyDictionary<string, object>>();
+
+            // For local master key
+            var localMasterKey = File.ReadAllText("master-key.txt");
+            var localMasterKeyBytes = Convert.FromBase64String(localMasterKey);
+
+            var localOptions = new Dictionary<string, object>
+            {
+                { "key", localMasterKeyBytes }
+            };
+            kmsProviders.Add("local", localOptions);
+
+            /* For Aws KMS, uncomment this block.
+            var awsAccessKey = "<Aws access key>";
+            var awsSecretAccessKey = "<Aws secret access key>";
+            var awsKmsOptions = new Dictionary<string, object>
+            {
+                { "accessKeyId", awsAccessKey },
+                { "secretAccessKey", awsSecretAccessKey }
+            };
+            kmsProviders.Add("aws", awsKmsOptions);
+            */
+
+            /* For Azure KMS, uncomment this block.
+            var azureTenantId = "<Azure account organization>";
+            var azureClientId = "<Azure client ID>";
+            var azureClientSecret = "<Azure client secret>";
+            var azureKmsOptions = new Dictionary<string, object>
+            {
+                { "tenantId", azureTenantId },
+                { "clientId", azureClientId },
+                { "clientSecret", azureClientSecret }
+            };
+            kmsProviders.Add("azure", azureKmsOptions);
+            */
+
+            /* For Gcp KMS, uncomment this block.
+            var gcpEmail = "<Gcp email>";
+            var gcpPrivateKey = "<Gcp private key>";
+            var gcpKmsOptions = new Dictionary<string, object>
+            {
+                { "email", gcpEmail },
+                { "privateKey", gcpPrivateKey }
+            };
+            kmsProviders.Add("gcp", gcpKmsOptions);
+            */
+
+            var schemaMap = new Dictionary<string, BsonDocument>();
+            schemaMap.Add(medicalRecordsNamespace.ToString(), schema);
+
+            var extraOptions = new Dictionary<string, object>()
+            {
+                /* 
+                 *   uncomment the following line if you are running mongocryptd manually
+                { "mongocryptdBypassSpawn", true }
+                */
```

Comment:
I found the comment structure confusing. I recommend:

                // uncomment the following line if you are running mongocryptd manually
                // { "mongocryptdBypassSpawn", true }